game.js

```diff
@@ -29,6 +31,14 @@ class Game extends Screen {
 		this.incorrectGuessChar = null
 		this.pauseUntilMilliSecond = 0 // The # of ms since the program started to pause until
 		this.puzzleRevealCountdown = 0
+    this.buttonReturnMenu = createButton('Return to menu')
+    this.buttonReturnMenu.hide()
+    this.pointsToGo
```

Comment:
You can just calculate this when you need it (as a function) rather than storing it and keeping it up-to-date.